Jenkinsfile

```diff
@@ -22,7 +22,8 @@ def cmake_build(Map conf=[:]){
     def debug_flags = "-g -fno-omit-frame-pointer -fsanitize=undefined -fno-sanitize-recover=undefined " + conf.get("extradebugflags", "")
     def build_envs = "CTEST_PARALLEL_LEVEL=4 MIOPEN_CONV_PRECISE_ROCBLAS_TIMING=0 " + conf.get("build_env","")
     def prefixpath = conf.get("prefixpath","/usr/local")
-    def setup_args = " -DMIOPEN_GPU_SYNC=Off " + conf.get("setup_flags","")
+    def mlir_args = " -DMIOPEN_USE_MLIR=" + conf.get("mlir_build", "ON")
```

Comment:
Hmm... Is it really necessary?

Reply:
Since `MIOPEN_USE_MLIR` becomes the default therefore no, it isn't necessary. But at the same time the `mlir_build` option in general gives a individual stage the flexibility to turn of MLIR build if necessary. This might be useful if someone wants to use the CI to regress the non-MLIR scenario. 

Talking about which, do you think it is necessary to add a couple of smoke test that build without MLIR?

Either way I'm neutral to this... And can remove/keep based on your suggestion.

Reply:
Let's keep this.

Reply:
> do you think it is necessary to add a couple of smoke test that build without MLIR?

No. Thanks.